<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation</string>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="nextFrame_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prevFrame_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playPause_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Chosen Directory</string>
    </property>
    <widget class="QLineEdit" name="directory_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="directory_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="number_of_files_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of files</string>
     </property>
    </widget>
    <widget class="QLabel" name="number_of_files_value_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="format_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>File format</string>
     </property>
    </widget>
    <widget class="QLabel" name="format_value_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="header_file_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>281</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="directory_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Header File</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_File"/>
    <addaction name="actionLoad_File"/>
    <addaction name="actionLoad_Data"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Header_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuColors">
    <property name="title">
     <string>Colors</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionShow_3D_Model"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionShow_Range_Plot"/>
    <addaction name="actionShow_2D_layer_Plot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuColors"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuGraph"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="action_omf_file">
   <property name="text">
    <string>.omf file</string>
   </property>
  </action>
  <action name="action_ohf_file">
   <property name="text">
    <string>.ohf file</string>
   </property>
  </action>
  <action name="actionLoad_File">
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="actionLoad_Data">
   <property name="text">
    <string>Load Directory</string>
   </property>
  </action>
  <action name="actionShow_3D_Model">
   <property name="text">
    <string>Show 3D Model</string>
   </property>
  </action>
  <action name="actionShow_Range_Plot">
   <property name="text">
    <string>Show Range Plot</string>
   </property>
  </action>
  <action name="actionShow_2D_layer_Plot">
   <property name="text">
    <string>Show 2D layer Plot</string>
   </property>
  </action>
  <action name="actionLoad_Header_File">
   <property name="text">
    <string>Load Header File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
